dts/msm8916/msm8916-samsung-r02.dts

```diff
@@ -29,4 +29,20 @@
 		lk2nd,match-bootloader = "G530W*";
 		#include "msm8916-samsung-gprime.dtsi"
 	};
+	
+	/*
+	 * Before building for G357FZ, please comment out all dtbs except
+	 * "$(LOCAL_DIR)/msm8916-samsung-r02.dtb" at './rules.mk'.
+	 * heatqlte doesn't work with multi-dtbs; only goes into "Download mode".
+	 */
+
+	heatqlte {
+		model = "Samsung Galaxy Ace 4";
```

Comment:
Maybe add the model here so it is clear which device is meant:

```suggestion
		model = "Samsung Galaxy Ace 4 (SM-G357FZ)";
```

... assuming this fits on a single line on the lk2nd screen.